flake8_annotations_complexity/ast_helpres.py

```diff
@@ -9,6 +10,8 @@ def get_annotation_complexity(annotation_node, default_complexity: int = 1) -> i
         except (SyntaxError, IndexError):
             return default_complexity
     if isinstance(annotation_node, ast.Subscript):
+        if sys.version_info >= (3, 9):
+            return 1 + get_annotation_complexity(annotation_node.slice)
```

Comment:
Please, add test for this particular case.

Reply:
I've added this line because the existing tests are failing without it.  `annotation_node.slice.value` raises an AttributeError.  Adding a try-except doesn't work either, because in some cases `annotation_node.slice.value` doesn't raise any Exception, but use the wrong value, leading to different results which again, breaks the existing tests.

If you do have any particular tests in mind for this case, I'm happy to add them.

I could also split the single commit into 3 commits, pushing them one by one, to illustrate that the current implementation fails with Python 3.9 and that try-except doesn't work either.